src/main/java/pl/patrycja/ox/gameflow/Mode.java

```diff
@@ -54,4 +49,9 @@ private Player changeInitialPlayer(List<Player> players, Player initialPlayer) {
         }
         return initialPlayer;
     }
+
+    private Judge createJudge(List<Player> players) {
+        ScoreBoard scoreBoard = new ScoreBoard(players);
+        return new Judge(gameSettings, ui, scoreBoard);
+    }
```

Comment:
Czemu to jest niżej ? Według clean codu było dobrze :)

Reply:
poprawie